pkg/mimirtool/commands/alerts.go

```diff
@@ -109,9 +111,36 @@ func (a *AlertmanagerCommand) getConfig(_ *kingpin.ParseContext) error {
 		return err
 	}
 
-	p := printer.New(a.DisableColor)
+	if a.OutputDir == "" {
+		p := printer.New(a.DisableColor)
+		return p.PrintAlertmanagerConfig(cfg, templates)
+	}
+	return a.outputAlertManagerConfigTemplates(cfg, templates)
+}
 
-	return p.PrintAlertmanagerConfig(cfg, templates)
+func (a *AlertmanagerCommand) outputAlertManagerConfigTemplates(config string, templates map[string]string) error {
+	var baseDir string
+	var fileOutputLocation string
+	baseDir, err := filepath.Abs(a.OutputDir)
+	if err != nil {
+		return err
+	}
+	fileOutputLocation = filepath.Join(baseDir, "config.yaml")
+	log.Debugf("writing the config file to %s", fileOutputLocation)
+	err = os.WriteFile(fileOutputLocation, []byte(config), os.FileMode(0o600))
+	if err != nil {
+		return err
+	}
+
+	for fn, template := range templates {
+		fileOutputLocation = filepath.Join(baseDir, fn)
+		log.Debugf("writing the template file %s", fileOutputLocation)
```

Comment:
```suggestion
		log.Debugf("writing the template file to %s", fileOutputLocation)
```

to match the message above

Reply:
Thanks for your review. I was quite stuck on how I would add a test since there isn't much to use as reference.  I have now added a test as you suggested, and I had to add a minimal valid alert manager config to `./testdata` for `cmd.readAlertManagerConfig()`. An invalid alert manager config will error out (although there isn't a test for that). I am also not entire sure about the naming conventions here on both the (test) function name and the variables, more than happy to change them along with any other suggestions.

Also added `to` to the debug line, not entirely sure how I have missed that initially.